operators/oadp-operator/0.4.0/manifests/veleroes.crd.yaml

```diff
@@ -0,0 +1,403 @@
+                          description: Specify whether the Secret or its key must
+                            be defined
+                          type: boolean
+                      required:
+                      - key
+                      type: object
+                    default:
+                      description: Default indicates this location is the default
+                        backup storage location.
+                      type: boolean
+                    objectStorage:
+                      description: ObjectStorageLocation specifies the settings necessary
+                        to connect to a provider's object storage.
+                      properties:
+                        bucket:
+                          description: Bucket is the bucket to use for object storage.
+                          type: string
+                        caCert:
+                          description: CACert defines a CA bundle to use when verifying
+                            TLS connections to the provider.
+                          format: byte
+                          type: string
+                        prefix:
+                          description: Prefix is the path inside a bucket to use for
+                            Velero storage. Optional.
+                          type: string
+                      required:
+                      - bucket
+                      type: object
+                    provider:
+                      description: Provider is the provider of the backup storage.
+                      type: string
+                    validationFrequency:
+                      description: ValidationFrequency defines how frequently to validate
+                        the corresponding object storage. A value of 0 disables validation.
+                      nullable: true
+                      type: string
+                  required:
+                  - objectStorage
+                  - provider
+                  type: object
+                type: array
+              customVeleroPlugins:
+                items:
+                  properties:
+                    image:
+                      type: string
+                    name:
+                      type: string
+                  required:
+                  - image
+                  - name
+                  type: object
+                type: array
+              defaultVeleroPlugins:
+                description: Plugin configuration
+                items:
+                  type: string
+                type: array
+              enableRestic:
+                description: Restic options. When false or missing, operator will
+                  attempt to delete Restic DaemonSet
+                type: boolean
+              noobaa:
+                description: Noobaa is a boolean to specify if we should install backup
+                  storage from OCS operator with Noobaa
+                type: boolean
+              olmManaged:
```

Comment:
*cough* update *cough*

Reply:
There have been quite a few API and description changes. I have told some folks run `make install` to install CRDs manually..